test/cql-pytest/test_describe.py

```diff
@@ -1603,16 +1606,16 @@ def test_desc_roles_with_passwords(cql):
         role_with_pass = "alice"
 
         create_stmt_without_pass = f"CREATE ROLE {role_without_pass} WITH LOGIN = false AND SUPERUSER = false;"
-        create_stmt_with_pass = f"CREATE ROLE {role_with_pass} WITH PASSWORD = 'some_funky_password'"
+        create_stmt_with_pass = f"CREATE ROLE {role_with_pass} WITH PASSWORD = '$2a$10$F5pRau9mKg5abP.DsuPQl.8rQpEoNm3OV91mKjb9vdKPUPejIPq/u'"
```

Comment:
I think you made a copy-pasto here - this is PASSWORD, not HASHED PASSWORD, so no need for that funny `$2$...` stuff here.

Reply:
Good catch, thank you!